deny.toml

```diff
@@ -0,0 +1,46 @@
+[bans]
+
+[bans.workspace-dependencies]
+
+unused = "deny"
+
+[licenses]
+
+allow = [
+    "MIT",
+    "Apache-2.0",
+    "ISC",
+    "Zlib",
+    "CC0-1.0",
+    "MPL-2.0", # TODO try and remove when we replace wasmedge_quickjs
+    "Unicode-3.0"
+]
+
+[sources]
+
+allow-git = [
+    "https://github.com/demergent-labs/wasmedge-quickjs",
+    "https://github.com/demergent-labs/candid",
+]
+
+unknown-git = "deny"
+
+unknown-registry = "deny"
+
+allow-registry = ["https://github.com/rust-lang/crates.io-index"]
+
+[advisories]
```

Comment:
Do you not think we should have all of these?

```
vulnerability = "deny"
notice        = "deny"
unsound       = "deny"
unmaintained  = "deny"
yanked        = "deny"
```

Reply:
And what about this?

```
[output]                      # fail on any non-deny finding
deny = ["warnings"]
```

Reply:
unmaintained and yanked are already in there
As per the docs https://embarkstudios.github.io/cargo-deny/checks/advisories/cfg.html
vulnerability, notice, and unsound have been removed and now emit errors.

I don't what that output is trying to do, nor do I see further elaboration of it in the docs https://embarkstudios.github.io/cargo-deny/checks/cfg.html?highlight=output#the-output-field-optional